sai.env

```diff
@@ -1,13 +1,13 @@
 # The sonic-swss-common and sonic-sairedis commits were taken from
-# sonic-buildimage master as of May 22, 2023
+# sonic-buildimage master as of Aug 15, 2023
 #
-# https://github.com/sonic-net/sonic-buildimage/tree/c5f2a0e
+# https://github.com/sonic-net/sonic-buildimage/tree/4acaaf8
 
-SWSS_COMMON_ID=5f41f0d
-SAIREDIS_ID=606703e
+SWSS_COMMON_ID=be425ed
+SAIREDIS_ID=eb24302
 
 # SAI version:
 #   Branch v1.12
 #   May 17, 2023
 
-SAI_ID=92875f9
+SAI_ID=ad12e9e
```

Comment:
why this has been changed?

Reply:
SAI latest commit on v1.12 branch is https://github.com/opencomputeproject/SAI/commit/ad12e9eac34996501691d7a159391e25d4353c46

Reply:
Should be the same commit as used in SONiC master